enterprise/models.py

```diff
@@ -1366,6 +1377,34 @@ def __repr__(self):
         return self.__str__()
 
 
+@python_2_unicode_compatible
```

Comment:
Good news!  We don't support python 2 anymore, so you don't need this decorator.

Reply:
Oh nice, I wasn't sure if we were building things to be backwards compatible or not. Good to know we aren't :D